Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM node:carbon
+#FROM node:dubnium
+
+WORKDIR /opt
+
+# Update apt-repo list and install prerequisites
```

Comment:
@Zooloo2014 I'm not sure if there is a need to do a `git clone` since the docker CI will be integrating with github.

Reply:
@de-laz But the docker CI just triggers the build; once it's triggered, Docker hub just runs the Dockerfile to build the image (I'm assuming this). 
How else will Docker Hub get the source to add to the image?

Reply:
How this will be setup in Dockerhub is to tell dockerhub where the Dockerfile is located and will be executed from there. We dont need to include cloning of any repos as it will have the correct references when building